pyramid_oereb/lib/records/plr.py

```diff
@@ -27,16 +27,17 @@ class PlrRecord(EmptyPlrRecord):
     Public law restriction record.
     """
 
-    def __init__(self, theme, information, law_status, published_from, responsible_office, symbol,
-                 view_service, geometries, sub_theme=None, type_code=None,
-                 type_code_list=None, basis=None, refinements=None, documents=None, info=None, min_length=0.0,
+    def __init__(self, theme, legende_text, law_status, published_from, published_until, responsible_office,
```

Comment:
Here we the LegendEntryRecord should be passed instead of the legend_text

Reply:
To keep it like the other record relations. You can access the legend text of LegendEntryRecord inside the init method to retrieve the content dict.

Reply:
I do not completely understand: is this wrong (also in the master https://github.com/openoereb/pyramid_oereb/blob/f148d34fa8bf29b1504c0d6e34aa5212b93578db/pyramid_oereb/standard/sources/plr.py#L309)?

Reply:
I canged the code like I mean it. Please have a look.